index.bs

```diff
@@ -963,6 +986,35 @@ method is invoked, the user agent MUST:
     Resolving this with good definitions or some other means will be addressed by resolving
     [Issue #613](https://github.com/w3c/webauthn/issues/613).
 
+    1. If |options|.{{PublicKeyCredentialRequestOptions/requireUserVerification}} is set to
+        {{UserVerificationRequirement/required}} and the |authenticator| is not capable of performing [=user verification=],
+        [=iteration/continue=].
+
+    1. Let |userVerification| be the <dfn>effective user verification requirement for assertion</dfn>, a Boolean value, as
+        follows. If <code>|options|.{{AuthenticatorSelectionCriteria/requireUserVerification}}</code>
+
+            <dl class="switch">
```

Comment:
Below procedure is not correct. 

There are two ways user verification can be requested for getAssertion. One is via `UV` flag to the authenticator if authenticator says that it supports `UV` in authenticatorGetInfo. Second is via ClientPin (also indicated via authenticatorGetInfo ) semantics where platform does the ClientPin flow and then in the authenticatorGetAssertion request passes `pinAuth` parameter. That `pinAuth` parameter is equivalent of asking "user verification" from such devices.

Reply:
I don't think we should explicitly couple these steps to CTAP. I think it's probably better to specify that "UV should be used" but leave unspecified how it should be done as that will be different for each backend protocol. CTAP may have two ways to trigger UV, but they're both conceptually the same thing in the WebAuthn abstraction. As long as we're compatible with the CTAP procedures we should be good, right?